/var/spool/slurmd/job4990909/slurm_script: line 108: --data_dir: command not found
Traceback (most recent call last):
  File "/home/l.peiwang/Master/PASTA/new_process.py", line 5, in <module>
    os.makedirs(output_dir, exist_ok=True)
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/scratch'
